lab-server/CONTRIBUTING.md

```diff
@@ -44,10 +44,10 @@ by using the following command: `docker compose --profile local up`
         "access-type" should be "bearer only". In the credentials tab you need
         to create a secret and update the value `KEYCLOAK_SECRET` accordingly
       - For the frontend the name should be "pharme-app". The "access-type"
-        should be set to "public"
+        should be set to "public" and the redirect URI should be `*`.
     - Create a user for testing (you can choose username and password freely, no
       roles are required)
-    - Set redirect URIs to `*` for both clients
+      - Remove the Required User Action `update password`
```

Comment:
This has never come up for me before (in the ~10+ time I have had to setup a user). Are you sure that you didn't accidentally click some field when creating the user?

Reply:
@dieknolle3333

Reply:
Status on this? @jannis-baum @dieknolle3333

Reply:
it happens to me as soon as I set a password. See the screenshots: Required user action isn't there right after creating the user, I set a password, then it's there. Feel like it's meant to be that way, but I needed to remove the action before proceeding
![password1](https://user-images.githubusercontent.com/57364771/169808940-303b3f0f-74a0-479d-95c6-3800d4528aa7.png)
![password2](https://user-images.githubusercontent.com/57364771/169808935-cf4523ab-185c-4eb5-b667-1127decc0205.png)
![password3](https://user-images.githubusercontent.com/57364771/169809023-77c29868-65d1-4e4f-bc84-0f5415c3f107.png)

Reply:
I am pretty sure this is happening because when you set the user's password you are not unchecking the `temporary` box. If the box is left checked, then the system expects the user to create a new one (for example by going through a password reset flow sent to them by email), hence the required action. However, since this is a non-existant user we want to set their password for them.

Either way I can see how this may be unclear. I see two possible ways to clear this up:

1. Leave the guide as is, keeping the part mentioning to uncheck the required user actions
2. Or update the documentation to mention explicitely unchecking the temporary box when settings the user's password

I will leave this up to you as I don't have a strong preference either way. Feel free to resolve this comment and re-request a review one you are done.